src/Parser.js

```diff
@@ -78,22 +80,28 @@ export function parse (value, points, declarations, rules, rulesets) {
 
 				index = length = offset = 0, ampersand = 1, type = temporary = ''
 				break
+			// -
+			case 45:
+				switch (previous) {
```

Comment:
`if` should be ok, as it's shorted.

Reply:
k, i thought `switch` is preferable, i thought there might be some perf magic going on in v8 for switch statements, gonna change this tomorrow